app/build.gradle

```diff
@@ -46,6 +46,8 @@ dependencies {
     implementation "com.mikepenz:aboutlibraries:7.1.0"
     implementation 'androidx.lifecycle:lifecycle-extensions:2.2.0'
     implementation 'androidx.preference:preference-ktx:1.1.1'
+    implementation "androidx.activity:activity-ktx:1.2.0-beta01"
```

Comment:
Would it be possible to bundle a stable version, not a beta?

Reply:
There isn't currently a stable version that provides the enhanced request permission / start activity for result flow. I don't think shipping a beta here is likely to be problematic

Reply:
Then perhaps add a comment stating this? So next time I can bump to a stable version and stick to stable releases. Also, do we need a beta from androidx.fragment:fragment-ktx as well?

Reply:
Yeah, unfortunately, it seems you do need both as the activity inherits from `FragmentActivity` and the functionality breaks without both being imported. Not sure if that will change in the future, but I will mention that also in my comment